src/net/sourceforge/kolmafia/persistence/SkillDatabase.java

```diff
@@ -916,11 +896,15 @@ public static final long birdSkillCasts(long availableMP) {
   }
 
   public static final long stackLumpsCost() {
+    return stackLumpsCost(Preferences.getInteger("_stackLumpsUses"));
```

Comment:
I broke this up because I could not figure out how to Mock Preferences.getInteger and this was so much easier to test.